libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -1469,19 +1472,52 @@ private static void addVideoUploadingClassIfMissing(AztecAttributes attributes)
         attributes.setValue(ATTR_CLASS, attrs.getAttributes().getValue(ATTR_CLASS));
     }
 
+    private static Attributes getElementAttributes(Spanned content, AztecText.AttributePredicate predicate) {
```

Comment:
Do you think we should find a way to merge this method with the one below  `getAllElementAttributes` to not repeat code? Maybe get all elements and just return the first of the list? (Maybe this is not wise performances side :))